homeassistant/components/openweathermap/weather_update_coordinator.py

```diff
@@ -178,19 +182,19 @@ def _convert_forecast(self, entry):
     def _get_rain(rain):
         """Get rain data from weather data."""
         if "all" in rain:
-            return round(rain["all"], 0)
+            return round(rain["all"], 2)
         if "1h" in rain:
-            return round(rain["1h"], 0)
+            return round(rain["1h"], 2)
         return "not raining"
```

Comment:
Would it make sense to return `0` instead of a string since the unit of measurement for this sensor is millimeters? To me at least, `0 mm` makes more sense than `not raining mm`.

Reply:
It makes more sense to me as well. However, like in the above comment, I'm trying to not be too disruptive. The integration makes checks for those strings in other places, and I wanted to be careful not to break anything.